src/packages/engine-core/src/NodeEngine.ts

```diff
@@ -438,15 +450,18 @@ ${chalk.dim("In case we're mistaken, please report this to us 🙏.")}`)
   }
 
   private async getEngineEnvVars() {
-    debug(`port: ${this.port}`)
-
     const env: any = {
       PRISMA_DML_PATH: this.datamodelPath,
-      PORT: String(this.port),
       RUST_BACKTRACE: '1',
       RUST_LOG: 'info',
     }
 
+    if (!this.useUds) {
+      env.PORT = String(this.port)
+    } else {
+      debug(`port: ${this.port}`)
```

Comment:
Why is the debug only for useUds === true?

Reply:
Oops that's wrong, will fix